app/components/footer_component.html.erb

```diff
@@ -2,14 +2,6 @@
   <%= render Footer::TalkToUsComponent.new %>
 <% end %>
 
-<% if feedback? %>
-  <%= render Footer::FeedbackComponent.new %>
-<% end %>
-
-<% if mailing_list? %>
```

Comment:
@martyn-w Just wanted to check why we are removing this mailing_list reference - is it not used?

Reply:
Hi @gemmadallmandfe, the mailing list component in the footer was removed in March 2023 (see PR https://github.com/DFE-Digital/get-into-teaching-app/pull/3196) - however this little piece of code was accidentally missed in that PR. Fortunately the logic continued to work correctly as the default behaviour in ruby is to treat an uninitialised `@instance` variable as null, rather than throw an error.

Removing this code just cleans up an earlier oversight; it should have no effect on the website.